### ISSUE TITLE
Remove DbusActivatable from desktop file

### ISSUE THREAD

**Issue by gnunn1** [gnunn1/terminix#345]
I'm creating this issue to ask for opinions on removing DBusActivatable=true. Frankly I'm thinking that DBusActivatable is causing me more issues and time wasted then it is worth. Some of the issues caused so far by DBusActivatable, either directly or indirectly, include:

* Unity starts Terminix in the root directory, requires a workaround to fix
* Gnome Shell 3.14 starts terminix in the root directory, requires a workaround to fix
* Workaround for previous two issues has caused issues in other places and it feels like I'm constantly having to tune it

So in a nutshell, I'm thinking of making the following changes:

* Set DBusActivatable to false in the desktop file by default
* Remove the workaround code in app.d for fixing Unity and earlier Gnome Shell versions
* When launching terminix fresh with no parameters, the initial shell directory shall be CWD. If a launcher is doing something funky it's a bug with the launcher and I'm not working around it.

**Comment by gnunn1** [comment]
Looks like my last workaround fixed all the issues, closing...for now. :)
